### DISCUSSION TITLE
color palette similar to ore or golem for all items made of iron

### DISCUSSION THREAD

**Discussion by TheGreenGuy7531** [BedrockTweaks/Files#145] [Ideas]
![Screenshot_20240708-171539](https://github.com/BedrockTweaks/Files/assets/175058715/a7942090-a3c3-4156-8738-56d93be9fbc0)
![Screenshot_20240708-171444](https://github.com/BedrockTweaks/Files/assets/175058715/a58a22b4-273e-408d-aa0a-e0268a831932)

**Comment by SquatchHunter** [comment]
interesting, one for the "bedrock exclusive" list for sure.
no plans for it for now but I like the concept
